partner_invoicing_mode/models/res_company.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    invoicing_mode_standard_last_execution = fields.Datetime(
+        string="Last execution (standard)",
+        help="Last execution of standard invoicing.",
+        readonly=True,
+    )
```

Comment:
I don't understand what this serves?

You can configure the cron job interval.
If you want a weekly or fortnightly invoicing on a fixed day, you can configure the cron to run on that specific day with an interval of respectively 7 or 14 days.
Otherwise you can configure the cron to run on a daily basis and the partner `next_invoice_date` will ensure you don't invoice more often than the interval.

So what brings the `invoicing_mode_*_last_execution` company value ?

Reply:
I think it's done at purpose to avoid duplicate runs. @TDu You can confirm this ?

Reply:
ping @TDu @sebalix